config/jobs/kubernetes/k8s-infra-public-pii/k8s-infra-public-pii.yaml

```diff
@@ -0,0 +1,34 @@
+periodics:
+  - interval: 168h
+    agent: kubernetes
+    name: public-log-vendor-matcher
+    annotations:
+      testgrid-dashboards: k8s-infra-public-pii-all
+      testgrid-tab-name: public-log-vendor-matcher
+      testgrid-alert-email: public-log-vendor-matcher@ii.coop
+      description: Runs the public-log-vendor-matcher job for generating data that is used for reviewing Kubernetes Public artifact traffic in DataStudio
+    decorate: true
+    spec:
+      containers:
+        - name: public-log-vendor-matcher
+          image: public-log-vendor-matcher #something
+          command:
+            - /usr/local/bin/docker-entrypoint.sh
+            - postgres
+          imagePullPolicy: Always
+          volumeMounts:
+            - name: gcp-app-creds
+              mountPath: /etc/asn-etl-pipeline
+          env:
+            - name: GCP_PROJECT
+              value: k8s-infra-ii-sandbox
+            - name: GCP_SERVICEACCOUNT
+              value: asn-etl@k8s-infra-ii-sandbox.iam.gserviceaccount.com
+            - name: GCP_BIGQUERY_DATASET
+              value: etl_script_generated_set
+            - name: GOOGLE_APPLICATION_CREDENTIALS
+              value: /etc/asn-etl-pipeline/asn-etl-pipeline-gcp-sa.json
```

Comment:
We are definitely not passing around service account keys as secrets, we should be using workload identity instead